socorro/processor/cache_manager.py

```diff
@@ -266,7 +296,7 @@ def _event_generator(self, nonblocking=False):
         self.running = True
         processed_events = False
         num_unhandled_errors = 0
-        last_usage_update = datetime.datetime.now()
+        next_heartbeat = time.time() + 1
```

Comment:
`time.time()` returns seconds since the epoch as a float with fractional seconds. We add 1 here to determine the next heartbeat and debounce until we hit that.